<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWidget</class>
 <widget class="QWidget" name="EmployeeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/Resource_ha.qrc">
    <normaloff>:/F1/fromHelyx/mainicon.svg</normaloff>:/F1/fromHelyx/mainicon.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QTableWidget" name="employeeStatustableWidget">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>73</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Empleado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ausencias</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Licencias</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Restantes</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="giveDays_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Dar Días</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/pencil16.png</normaloff>:/F1/fromHelyx/pencil16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="descountDays_pushButton">
       <property name="text">
        <string>Descontar Días</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/win_closeAll16.png</normaloff>:/F1/fromHelyx/win_closeAll16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QCalendarWidget" name="employeecalendarWidget"/>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="restricciones_pushButton">
       <property name="text">
        <string> Restricciones</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/padlock.png</normaloff>:/F1/fromHelyx/padlock.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="feriados_pushButton">
       <property name="text">
        <string> Feriados</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/christmas_day.png</normaloff>:/F1/fromHelyx/christmas_day.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="period_pushButton">
       <property name="text">
        <string>Períodos</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/favouritesAdd16.png</normaloff>:/F1/fromHelyx/favouritesAdd16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="employee_pushButton">
       <property name="text">
        <string> Empleados</string>
       </property>
       <property name="icon">
        <iconset resource="../images/Resource_ha.qrc">
         <normaloff>:/F1/fromHelyx/employees.png</normaloff>:/F1/fromHelyx/employees.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/Resource_ha.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>employeecalendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>showDaysByEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>giveDays_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>giveDays()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descountDays_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>giveDays()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>feriados_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>adminFeriados()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restricciones_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>adminRestricciones()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>period_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>newPeriod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>employee_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>EmployeeWidget</receiver>
   <slot>adminEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showDaysByEmployee()</slot>
  <slot>giveDays()</slot>
  <slot>adminFeriados()</slot>
  <slot>adminRestricciones()</slot>
  <slot>newPeriod()</slot>
  <slot>adminEmployee()</slot>
 </slots>
</ui>
